cypress/integration/common/selectors-and-assertions.spec.ts

```diff
@@ -23,6 +23,16 @@ export const givenIHaveACookieSetAndIVisit = (url: string): void => {
   cy.visit(url);
 };
 
+export const givenIHaveACookieSetAndHaveSignedInIVisit = (
+  url: string
+): void => {
+  cy.setCookie("submissionId", v4());
+  cy.setCookie("next-auth.session-token", Cypress.env("SESSION_TOKEN"), {
```

Comment:
Instead of testing that signing in with B2C works, I've gotten a session token that expires in 2031 that can be used locally and in the pipeline for the e2e tests.